_components/checkbox/checkbox.md

```diff
@@ -10,16 +10,8 @@ permalink: /components/checkbox/
 redirect_from:
 - /form-controls/02-checkboxes/
 subnav:
-- text: Preview
-  href: '#checkbox-preview'
-- text: Code
-  href: '#checkbox-code'
-- text: Guidance
-  href: '#checkbox-guidance'
-- text: Package
-  href: '#checkbox-package'
-- text: Latest updates
-  href: '#changelog'
+- text: checkbox accessibility tests
```

Comment:
@RachelCorsino 

**issue (blocking):** Little typo
```suggestion
- text: Checkbox accessibility tests
```

Reply:
Resolved in [5808ba8](https://github.com/uswds/uswds-site/pull/2803/commits/5808ba81a1e73b90ad5129c8a440f7a27c6ad4a9)